packages/dd-trace/src/plugins/plugin.js

```diff
@@ -26,10 +26,23 @@ class Subscription {
 }
 
 module.exports = class Plugin {
-  constructor (tracer) {
+  constructor (tracer, tracerConfig) {
```

Comment:
Passing this here means that if the tracer config is updated, the update won't propagate anymore.

Reply:
tracerConfig is an object, so wouldn't modifying properties of that object propagate everywhere?